### DISCUSSION TITLE
Use Apache with traefik or traefik in the NC container

### DISCUSSION THREAD

**Discussion by erwanlpfr** [nextcloud/all-in-one#2471] [Questions] [answered]
Hello,

I am actually facing poor performance while running NC AIO (manual install) with traefik.
All of my traffic is handled by the linked Apache configuration from AIO.
I get loading time up to 90 seconds to load 15 Mo or common-core.js. Sometimes it is worse.
Also, the upload speed is low.

So, I guess It is from my settings.

Do you advise or not to take out the Apache CT and redirect the traefik to NC CT directly.
Or did I miss something about the usage of this apache?

EDIT : according to iperf3, traefik releases a perfect bandwidth. So, the limitation is between traefik/apache/nextcloud.

Thanks for you work!

**Comment by szaimen** [comment, marked as answer]
Hi, can you follow https://github.com/nextcloud/all-in-one/discussions/2475?

**Reply by erwanlpfr** [reply]
I will ! I read it quickly and it makes sense, I'll try it !
